publish-packages.cmd

```diff
@@ -0,0 +1,28 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+set packagesLog=publish-packages.log
+echo Running publish-packages.cmd %* > %packagesLog%
+
+if /I [%1] == [/?] goto Usage
+if /I [%1] == [/help] goto Usage
+
+REM ensure that msbuild is available
+echo Running init-tools.cmd
+call %~dp0init-tools.cmd
+
+echo msbuild.exe %~dp0src\publish.builds %* /nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog% >> %packagesLog%
+call msbuild.exe %~dp0src\publish.builds %* /nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog%
+if NOT [%ERRORLEVEL%]==[0] (
+  echo ERROR: An error occurred while publishing packages, see %packagesLog% for more details.
+  exit /b
+)
+
+echo Done publishing packages.
+exit /b
+
+:Usage
```

Comment:
I think in this case the error message will be "Missing property CloudDropAccountName", @jhendrixMSFT is this correct?
We could add to it that this is used only by the build pipeline...